apps/style/curriculum/certificates.scss

```diff
@@ -0,0 +1,6 @@
+#certificate-batch {
+  p {
+    font-size: inherit;
+    line-height: inherit;
```

Comment:
this is to override these styles: https://github.com/code-dot-org/code-dot-org/blob/c1efb7249b926a1691e23717dfdeff02f04f81e5/dashboard/app/assets/stylesheets/scaffolds.scss#L10-L12

without this, `SafeMarkdown` renders `p` elements whose font and line size can not be overridden.